quickleft/react-loader
# Added support for delay prop.

#### Supports delay property to delay showing spinner

Very often a spinner is used for loading remote content, typically after filling in a form (e.g. login or similar). This PR adds support for a `delay` property which delays how soon a spinner would replace the content beneath it.

#### Where should the reviewer start?

At the beginning! Really, it's a small module and small change.

#### How should this be manually tested?

The only way to test currently is to add a delay prop to the Loader module. For instance a delay of 300 would mean that a spinner would not be shown, unless the operation takes more than 300 milliseconds. If it does, the spinner would show normally after 300 milliseconds have passed.

Since it is a small change and I have limited experience with phantomjs, I did not add the necessary async mojo to the tests to be able to verify the correct operation.
